docs/config.md

```diff
@@ -653,6 +653,18 @@ perf_analyzer_flags:
     latency-report-file: /path/to/latency/report/file
 ```
 
+The `input-data`, `shape`, and `streaming` perf_analyzer options are additive and can take either a single string (non-list) or a list of strings:
+
+```yaml
+...
```

Comment:
Let's keep the examples consistent and not use dots anywhere.

Reply:
updated 👍